input/pages/negation.md

```diff
@@ -0,0 +1,217 @@
+
+"note" : ...,
+
+"dosage" : ...
+
+}
+```
+
+#### Documenting no members of an entire value set performed for a given reason.
+
+This can occur when the no treatment of the type included in the value set is appropriate. The approach provided allows systems to document using one profiled data instance that none of the activities in a particular value set were performed, rather than requiring documentation of multiple individual activities from the value set.
+
+The following example documents that providers did not prescribe any of the medications in the “Antithrombotic Therapy for Ischemic Stroke” value set using the <a href="StructureDefinition-qicore-notDoneValueSet.html">notDoneValueSet</a> extension fulfills criteria for the numerator:
+
+**NOTE:** Implementing systems must ensure that this approach does not result in conflicting data. For example, the above example indicating no administration of a medication in the Antithrombotic Therapy value set should not be used if there are administrations of individual medications in the same value set. In other words, it is a contradiction to say “a provider administered a specific medication” at the same time as “a provider did not administer any of the medications in this value set” if that value set includes the medication that was administered in the specific case.
+
+See the <a href="MedicationAdministration-negation-example.html">MedicationAdministration example using a value set</a> for a complete example.
+
+```json
+{
+
+"resourceType" : "MedicationAdministration",
+
+"id" : "negation-example",
+
+"meta" : {
+
+"profile" : ["http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-medicationadministrationnotdone"]
+
+},
+
+"extension" : [{
+
+"url" : "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-recorded",
+
+"valueDateTime" : "2015-01-15"
+
+}],
+
+"status" : "not-done",
+
+"statusReason" : [{
+
+"coding" : [{
+
+"system" : "http://snomed.info/sct",
+
+"code" : "183966005",
+
+"display" : "Drug treatment not indicated (situation)"
+
+}]
+
+}],
+
+"medicationCodeableConcept" : {
+
+"extension" : [{
+
+"url" : "http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-notDoneValueSet",
+
+"valueCanonical" : "http://cts.nlm.nih.gov/fhir/ValueSet/2.16.840.1.113762.1.4.1110.62"
+
+}],
+
+"text" : "Not Done Value Set: Antithrombotic Therapy for Ischemic Stroke"
+
+},
+
+"subject" : ...,
+
+"context" : ...,
+
+"supportingInformation" : ...,
+
+"effectivePeriod" : ...,
+
+"request" : ...,
+
+"note" : ...,
+
+"dosage" : ...
+
+}
+```
+
+### Negation in CQL
+
+For quality measurement and reporting, measure expression may only need to determine the existence or absence of an activity or event to determine if criteria have been met. If the reason for absence is not relevant to the measure evaluation, the absence of evidence pattern should be used as described in the [Using CQL section of the Quality Measure IG](https://hl7.org/fhir/us/cqfmeasures/using-cql.html#negation-in-fhir).
```

Comment:
eventually point to http://hl7.org/fhir/uv/cql/2024Jan/using-cql.html#negation-in-fhir instead of QMIG

See also https://jira.hl7.org/browse/FHIR-43531

Reply:
I assume https://hl7.org/fhir/us/cqfmeasures/using-cql.html#negation-in-fhir will still work after than Jan 2024 ballot of QMIG is published, even if it moves from section 4.9.1 to 4.10 (per FHIR-43531)